pipelines/upgrade/06-proxy_to_intermediate.yaml

```diff
@@ -25,4 +26,5 @@
     - role: remove_pulp2
       when:
         - forklift_upgrade_version_intermediate == "4.0"
+        - pipeline_remove_pulp2 is defined
```

Comment:
Are these evaluated in order every time? or is it safer to combine this with boolean check?

Reply:
in order.